libs/accounts/src/lib/accounts-table.spec.tsx

```diff
@@ -44,62 +44,79 @@ describe('AccountsTable', () => {
   });
 
   it('should apply correct formatting', async () => {
-    await act(async () => {
-      render(
-        <AccountTable
-          rowData={singleRowData}
-          onClickAsset={() => null}
-          isReadOnly={false}
-        />
-      );
-    });
+    const { container } = render(
+      <AccountTable
+        rowData={singleRowData}
+        onClickAsset={() => null}
+        isReadOnly={false}
+      />
+    );
+
     const cells = await screen.findAllByRole('gridcell');
     const expectedValues = ['tBTC', '1,256.00', '1,256.00', '2,512.00', ''];
     cells.forEach((cell, i) => {
       expect(cell).toHaveTextContent(expectedValues[i]);
     });
-    const rows = await screen.findAllByRole('row');
-    expect(rows.length).toBe(6);
+    const rows = container.querySelector('.ag-center-cols-container');
+    expect(rows?.childElementCount).toBe(1);
```

Comment:
We are losing some coverage here as we aren't checking the correct number of rows are rendered. Not really bothered though as thats testing the implementation of aggrid

Reply:
There is only one row rendered. Previously number of rows was checked by counting elements with `role="row"`  was 6 because there are multiple elements which have this row in whole table. 
Now we are checking exact number of rows, which should be 1